rlclientlib/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ set(PROJECT_SOURCES
   console_tracer.cc
   continuous_action_response.cc
   decision_response.cc
+  dedup.cc

```

Comment:
shouldn't these be added to the `vcxproj` file too?

Reply:
good point